pkg/minikube/cluster/cluster_darwin.go

```diff
@@ -69,7 +71,8 @@ func createXhyveHost(config MachineConfig) *xhyveDriver {
 		BootCmd:        "loglevel=3 user=docker console=ttyS0 console=tty0 noembed nomodeset norestore waitusb=10 base host=" + cfg.GetMachineName(),
 		DiskSize:       int64(config.DiskSize),
 		Virtio9p:       true,
-		Virtio9pFolder: "/Users",
+		QCow2:          false,
```

Comment:
What happens if we pass these new options to an older version of the xhyve driver? Will it just ignore them?